include/ultrahdr/jpegdecoderhelper.h

```diff
@@ -37,8 +37,8 @@ extern "C" {
 
 // constraint on max width and max height is only due to device alloc constraints
 // Can tune these values basing on the target device
-static const int kMaxWidth = 8192;
-static const int kMaxHeight = 8192;
+static const int kMaxWidth = 5312;
```

Comment:
this isn't a good idea to restrict the library, just to ensure fuzzer doesn't timeout.

16 MP is not large enough.